adbe/adb_enhanced.py

```diff
@@ -383,7 +383,8 @@ def handle_list_devices():
             len(device_infos) == 1 and len(device_infos[0]) == 0):
         print_error_and_exit('No attached Android device found')
     elif len(device_infos) == 1:
-        _print_device_info()
+        device_serial = device_infos[0].split(" ")[0]
```

Comment:
I can see a slight benefit of taking this approach. But am I missing something?
Any major advantage of this change?

Reply:
If the device ID is needed even if there is only one device.
I am migrating [AMDH](https://github.com/SecTheTech/AMDH) to use adbe, as the device ID is a needed information for the reports, I sent this pull request.

Reply:
Sounds good.